### PR TITLE
Update `captum_explainability` example to use `CaptumExplainer`

### PR DESCRIPTION
**Pull request by RBendias**

### CODE REVIEW

File: examples/explain/captum_explainer.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        value=200,
+    ),
+)
+
+# Compute the explanation:
+explanation = explainer(
+    x=data.x,
+    edge_index=data.edge_index,
+    index=index,
+)
```
Comment (wsad1) [inline examples/explain/captum_explainer.py:73]:
```suggestion
# Instantiate the CaptumExplainer algorithm:
algorithm = CaptumExplainer(attribution_method='IntegratedGradients', )

# Create an instance of the Explainer class with the GNN model
# and the CaptumExplainer algorithm:
explainer = Explainer(
    model=model,
    algorithm=algorithm,
    explanation_type='model',  # or phenomenon
    model_config=dict(
        mode='multiclass_classification',
        task_level='node',
        return_type='log_probs',
    ),
    node_mask_type='attributes',
    edge_mask_type='object',
    threshold_config=dict(
        threshold_type='topk',
        value=200,
    ),
)

# Select the output indices of nodes to be explained:
index = [10, 55]

# Compute the explanation:
explanation = explainer(
    x=data.x,
    edge_index=data.edge_index,
    index=index,
)
```
Lets explain multiple indicies to show the user, that its supported.

**Review by wsad1 — APPROVED** [review]

File: examples/explain/captum_explainability.py
Comment (wsad1) [inline examples/explain/captum_explainability.py:1]: We link to this example in the doc. https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.nn.models.to_captum_model.html#torch_geometric.nn.models.to_captum_model. 
Let update that. and also update the doc to say that `to_captum_model` is depreciated.

File: examples/explain/captum_explainer.py
Comment (wsad1) [inline examples/explain/captum_explainer.py:1]: Update this [doc](https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.explain.algorithm.CaptumExplainer.html#torch_geometric.explain.algorithm.CaptumExplainer) to point to this example .

**Review by wsad1 — APPROVED** [review]
Thanks!
Changes look good.

**Review by rusty1s — APPROVED** [review]

**Pull request merged**